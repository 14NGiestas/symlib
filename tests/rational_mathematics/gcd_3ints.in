          41          27          68
          36          61           2
          98          40          39
          63          23          36
          63          22          15
          79          22          44
          91          61          85
          67          13          93
           1          72          82
          38          18          53
          61          54          30
          51          55          81
          23          47          67
          74           6           1
          39          50          80
          47          50          34
          35          85           3
           4          67          57
          77          76          43
          68          30          28
          70          26          12
          61          71          56
          86          54          33
          39          52          46
          23          16          31
          81           5           2
          34          35          38
          14          86          57
          13          25          96
          68          31          19
          76          54          30
          59          13          17
          30           2          20
          83          42          46
          64          54          99
          29          45          12
          83          66           8
          63          98          36
          63          61           2
          48          29          77
           4          74          39
          78          17          18
          96          26          19
          34          40          97
          78          62           1
          83          17          53
          73           1          22
          13          29           3
          99           4          14
          39          93          25
          77          43           8
          48          91          71
          76          77          21
          94          11          85
          78          90          86
          64          22          32
          49          42          77
          21          78          97
          51          81           3
          65           9          54
          70         100           7
          87          98          51
          55          49          74
          10           7          31
          98          46          76
          40          97          81
          61          56          76
          34          64          59
          41          47          33
          99           7          23
          24          43          54
          12          52          13
          36          11          84
          90          51          51
          95          90          45
          26          90          65
          32          38          53
          55          69          17
          86          10          47
          13          81          33
          64          60          74
          91          16          39
          57          31          12
          99          91          98
          99          40          84
           6          87          58
           1          28          11
          89          59          86
           4          48          92
          50          69          46
          83          44          32
          32          75          93
          63          38          24
          21          64          67
          28          43          14
          16          43          12
          59          29          87
          62          86          48
          85          12          46
          57          73          13
