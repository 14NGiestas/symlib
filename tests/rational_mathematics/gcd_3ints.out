 # <fortpy version="1" template="integer"></fortpy>
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           2
           2
           1
           1
           1
           1
           1
           1
           1
           1
           1
           2
           1
           2
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           2
           2
           7
           1
           3
           1
           1
           1
           1
           1
           2
           1
           1
           1
           1
           1
           1
           1
           1
           3
           5
           1
           1
           1
           1
           1
           2
           1
           1
           1
           1
           1
           1
           1
           4
           1
           1
           1
           1
           1
           1
           1
           1
           2
           1
           1
